easybuild/easyconfigs/o/OpenMM/OpenMM-7.4.1_fix-Doxygen.patch

```diff
@@ -0,0 +1,42 @@
+fix build errors when using Doxygen 1.8.16 like:
```

Comment:
@boegel add author?

Reply:
@smoors I didn't create the patch, I basically just pulled it from the pull request I reference to below...